### ISSUE TITLE
Baleful

### ISSUE THREAD

**Issue by Nomad461** [arkayenro/arkinventory#729]
I would like to request a True/False variables in the rule that is True if an item is Baleful an False if it isn't.

**Comment by spiralofhope** [comment]
I am currently using this code:

``` lua
soulbound()
and useable()
and type( 'Armor' )
and tooltip( 'Baleful' )
```

To help streamline the code, I include specific item levels.  There's a second item level in there but I don't recall it offhand.

``` lua
(
  -- Baleful
      itemlevel( 650 )
  -- A lucky Baleful creation
  -- or itemlevel( TODO )
  -- Empowered Baleful
  or itemlevel( 695 )
)
and soulbound()
and useable()
and type( 'Armor' )
and tooltip( 'Baleful' )
```

**Comment by Nomad461** [comment]
Yeah, the patch that added 'Baleful' to the tooltip made this a lot easier.  For reference, the iLevel you're missing is 675.

**Comment by spiralofhope** [comment]
Thanks for mentioning that.

I also added ```lua uselevel( 100 )``` to further narrow it down.  Using ```lua tooltip()``` is a __really__ big performance hit.
